lib/components/LazyPicture.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <lazy-image v-bind="lazyImage">
+    <!-- eslint-disable-next-line vue/no-template-shadow -->
+    <template slot-scope="{ lazy }">
+      <picture v-if="lazy">
+        <source v-for="(source, index) in preparedSources" :key="index" v-bind="source">
+        <img :src="placeholder" :alt="$attrs.alt" :title="$attrs.title" loading="eager">
+      </picture>
+    </template>
+  </lazy-image>
+</template>
+
+<script>
+
+import { PLACEHOLDER } from '../utils/image'
+import LazyImage from './LazyImage'
+
+export default {
+
+  components: {
+    LazyImage
+  },
+
+  props: {
+
+    sources: {
+      type: Array,
+      default () {
+        return [
+          {
+            srcset: [],
+            type: null,
+            media: null
+          }
+        ]
+      }
+    }
+  },
+
+  data () {
+    return {
+      placeholder: PLACEHOLDER
```

Comment:
Wollen wir hier weiter mit einem transparenten Image arbeiten oder wollen wir das von außen reingeben können -> eine Art Fallback. Das könnte das Desktop Bild sein.

Reply:
Oder man nimmt hier einfach die Oberste `source`. Wäre in dem Fall Fallback Desktop.

Reply:
jupp können wir gern testen